v2/types.go

```diff
@@ -370,3 +392,13 @@ type UnbindRequest struct {
 type UnbindResponse struct {
 	// Currently, unbind responses have no fields.
 }
+
+// AlphaBindingRequest represents a request to do a GET on a particular binding.
+type AlphaBindingRequest struct {
+	// InstanceID is the ID of the instance the binding is for.
+	InstanceID string `json:"instance_id"`
+	// BindingID is the ID of the binding to delete.
+	BindingID string `json:"binding_id"`
+	// OriginatingIdentity is the identity on the platform of the user making this request.
+	OriginatingIdentity *AlphaOriginatingIdentity `json:"originatingIdentity,omitempty"`
```

Comment:
It's not clear to me yet what you should send in OriginatingIdentity for this request, is it clear to you @kibbles-n-bytes

Reply:
Hmm, I wasn't too familiar with originating identity when I put it here, but reading through the proposal, it seems like we probably don't need it. I suspect `GET`s will be driven by the platform, not the user.